tests/testImagePsfMatch.py

```diff
@@ -146,7 +146,7 @@ def testSubtractExposures(self):
 
         self.assertEqual(type(resultsAL.subtractedExposure), afwImage.ExposureF)
         self.assertEqual(type(resultsAL.psfMatchingKernel), afwMath.LinearCombinationKernel)
-        self.assertEqual(type(resultsAL.backgroundModel), afwMath.Function2D)
+        self.assertEqual(type(resultsAL.backgroundModel), afwMath.Chebyshev1Function2D)
```

Comment:
It was always returning Chebyshev1Function2D but SWIG showed the base class, not the actual class.